src/Core/Stack.php

```diff
@@ -232,11 +240,13 @@ public function getConfig()
      *
      * @since 1.2.0
      *
+     * @throws \RuntimeException
```

Comment:
What are the cases where it might throw a runtime exception? When documenting the @throws tags, adding that information adds a lot of value.

Reply:
Added